OpenRA.Mods.Common/Graphics/RectangleAnnotationRenderable.cs

```diff
@@ -0,0 +1,63 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Graphics;
+using OpenRA.Primitives;
+
+namespace OpenRA.Mods.Common.Graphics
+{
+	public struct RectangleAnnotationRenderable : IRenderable, IFinalizedRenderable
+	{
+		readonly WPos[] vertices;
+		readonly WPos effectivePos;
+		readonly int width;
+		readonly Color color;
+
+		public RectangleAnnotationRenderable(WPos diagVertice1, WPos diagVertice2, WPos effectivePos, int width, Color color)
```

Comment:
Please call these `tl` and `br` instead of `diagVertice1` and `diagVertice2`.

Reply:
OK